.devcontainer/devcontainer.json

```diff
@@ -2,7 +2,7 @@
     "name": "AstroPy",
     "image": "mcr.microsoft.com/devcontainers/miniconda:0-3",
     "onCreateCommand": "conda init bash && sudo cp .devcontainer/welcome-message.txt /usr/local/etc/vscode-dev-containers/first-run-notice.txt",
-    "postCreateCommand": "git fetch --tag && pip install tox && pip install -e .[all,test_all,docs]",
+    "postCreateCommand": "git fetch --all && pip install tox && pip install -e .[all,test_all,docs]",
```

Comment:
The correct option (from git's manpage) is `--tags`, not `--tag`.

Reply:
Makes sense, except creating codespaces *did* work at some point in the past. And now using `--all` is failing when running on the astropy repo while it worked on my local fork. Ugh.